Standards/scs-0100-v3-flavor-naming.md

```diff
@@ -271,6 +271,9 @@ It goes beyond the above example in checking that the discoverable
 features of flavors (vCPUs, RAM, Disk) match what the flavor names claim.
 This is used for SCS-compatible compliance testing.
 
+The web page <https://flavors.scs.community/>
+has a flavor name parser and (soon) generator.
```

Comment:
Maybe we should add that the functionality of the website is analogous to some of the scripts mentioned above (and in what way).

Reply:
"The flavor name parsing and (soon) flavor name generation of these scripts is also exposed via the web page <https://flavors.scs.community/>."